Sources/Snowplow/Network/DefaultNetworkConnection.swift

```diff
@@ -88,13 +88,20 @@ public class DefaultNetworkConnection: NSObject, NetworkConnection {
     public var serverAnonymisation = false
     private var dataOperationQueue = OperationQueue()
     private var builderFinished = false
+    /// Custom timeout for the requests
+    @objc
+    public var timeout: NSNumber
```

Comment:
Could you make this a private `let` property? As far as I can tell, changes to the `timeout` are not handled (the `urlSession` is not updated in case timeout changes) and letting users configure it in the constructor should be enough.